.changeset/old-boxes-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
```

Comment:
```suggestion
"@primer/react": minor
```

`minor` could be used here I think based on [versioning](https://github.com/primer/react/blob/main/contributor-docs/versioning.md#:~:text=A%20component%20is%20added) since we're adding a component.

Reply:
oops, thanksss